libraries/Microsoft.Bot.Builder/IReadWriteSlot.cs

```diff
@@ -7,7 +7,7 @@ namespace Microsoft.Bot.Builder
 {
     public interface IReadWriteSlot : IReadOnlySlot
     {
```

Comment:
I believe this interface should just be renamed to simply `ISlot`. An `ISlot` is implicitly fully read/write. Also see comments on `IFrame::AddSlot` as this is what would now be returned from the factory method and people wouldn't actually ever be constructing explicit `Slot` class instances any more.